libr/core/cmd_info.c

```diff
@@ -603,66 +603,52 @@ static int cmd_info(void *data, const char *input) {
 		case 't': // "it"
 			{
 				ut64 limit = r_config_get_i (core->config, "bin.hashlimit");
-				const char *fileName;
 				RBinInfo *info = r_bin_get_info (core->bin);
-				if (info) {
-					fileName = info->file;
-				} else {
-					int fd = r_io_fd_get_current (core->io);
-					RIODesc *desc = r_io_desc_get (core->io, fd);
-					fileName = desc? desc->name: NULL;
-				}
-				const bool is_json = input[1] == 'j'; // "itj"
-				RList *old_file_hashes = NULL;
-				if (!r_bin_file_hash (core->bin, limit, fileName, &old_file_hashes)) {
-					eprintf ("r_bin_file_hash: Cannot get file hashes");
-					r_list_free (old_file_hashes);
-					return 0;
-				}
-
-				if (old_file_hashes && r_list_empty (old_file_hashes)) {
-					// clean the old hashes list to reduce comparison operations in case it is allocated but empty
-					r_list_free (old_file_hashes);
-					old_file_hashes = NULL;
-				}
 				if (!info) {
-					info = r_bin_get_info (core->bin);
+					eprintf ("r_bin_get_info: Cannot get bin info\n");
+					return 0;
 				}
-				if (!info) {
-					eprintf ("r_bin_get_info: Cannot get bin info");
-					r_list_free (old_file_hashes);
+				RList *new_file_hashes = r_bin_file_hash (core->bin, limit);
```

Comment:
this does not re-compute the securihash, does it?

Reply:
You are right, it does not.